user-guide/src/SUMMARY.md

```diff
@@ -12,7 +12,7 @@
 - [Installation](./system-admins/installation/introduction.md)
   - [Dependencies](./system-admins/installation/dependencies.md)
   - [`branectl`](./system-admins/installation/branectl.md)
-  - [Control node](./system-admins/installation/control-node.md)
+  - [Central node](./system-admins/installation/control-node.md)
```

Comment:
I think this path has to be updated to `./system-admins/installation/central-node.md`, right?